From 74dc9181294e45cc4de90b40116369b76bbab7f6 Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@freescale.com>
Date: Wed, 26 Dec 2012 14:30:06 +0800
Subject: [PATCH 127/304] ENGR00238279 imx6: Add Email.apk into
 PRODUCT_PACKAGES

Default email apk in build.git is Email2.apk
which we do not have in source. So Email.apk
must be installed.

Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 imx6/imx6.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/imx6/imx6.mk b/imx6/imx6.mk
index 9846f99..7d0d50d 100644
--- a/imx6/imx6.mk
+++ b/imx6/imx6.mk
@@ -15,6 +15,7 @@ PRODUCT_PACKAGES += \
 	Gallery		    			\
 	SoundRecorder				\
         Camera                                  \
+	Email					\
 	FSLOta					\
 	VideoEditor				\
 	FSLProfileApp				\
-- 
1.8.0

